src/parser/smt2/Smt2.g

```diff
@@ -1414,6 +1414,7 @@ termNonVariable[cvc5::Term& expr, cvc5::Term& expr2]
  * (2) A string name.
  * (3) An expression expr.
  * (4) A type t.
+ * (5) A list of indices.
```

Comment:
In [ParserOp](https://github.com/cvc5/cvc5/blob/main/src/parser/parse_op.h#L39) (5) is called an operator object, and I think it meant field [d_op](https://github.com/cvc5/cvc5/blob/main/src/parser/parse_op.h#L72).
Perhaps (6) should be added to parser_op.h to point to field [d_indices](https://github.com/cvc5/cvc5/blob/main/src/parser/parse_op.h#L79) and used here.